src/oc_propagation_http_b3.erl

```diff
@@ -39,7 +39,7 @@ to_headers(#span_ctx{trace_id=TraceId,
                      trace_options=TraceOptions}) ->
     Options = case TraceOptions band 1 of 1 -> "1"; _ -> "0" end,
     %% iolist_to_binary only needed for versions before otp-21
-    EncodedTraceId = iolist_to_binary(io_lib:format("~32.16.0b", [TraceId])),
+    EncodedTraceId = iolist_to_binary(io_lib:format("~.16b", [TraceId])),
```

Comment:
I can either do this or check if the integer is between -9,223,372,036,854,775,808 and 9,223,372,036,854,775,807 in a guard. Thought this might be better?

Reply:
I can also do` binary:encode_unsigned(TraceId)` and check if its size 64 or 128 before formatting it. Let me know which approach you prefer.